examples/builder/builder-2/GenerateBuilderAttribute.cs

```diff
@@ -119,7 +118,7 @@ public override void BuildAspect(IAspectBuilder<INamedType> builder)
                 {
                     var isRequired = p.Attributes.OfAttributeType(typeof(RequiredAttribute))
                         .Any();
-                    var isInherited = p.DeclaringType != sourceType;
+                    var isInherited = !p.DeclaringType.Equals(sourceType);
```

Comment:
Why?

Reply:
It's generally better to compare `IType`s using `Equals()` than using reference equality.